lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -335,6 +363,8 @@ def test_mean_predictor_estimate_coefficients_nans(self):
 
         current_forecast_predictor = cube.collapsed(
             "realization", iris.analysis.MEAN)
+        current_forecast_predictor.data = (
```

Comment:
Why is this needed?

Reply:
This has been added because otherwise you get this error:
```python
test_EstimateCoefficientsForEnsembleCalibration.py:374: MaskedArrayFutureWarning: setting an item on a masked array which has a shared mask will not copy the mask and also change the original mask array in the future.
Check the NumPy 1.11 release notes for more information.
  current_forecast_predictor.data[0][0][0] = np.nan
```
As the array is being specifically set to have NaN value, the masked values have been filled. This is due to a feature of cube.collapsed where the output cube seems to always be a masked array regardless of whether the input cube is a masked array.